content/en/docs/instrumentation/net/automatic/config.md

```diff
@@ -0,0 +1,352 @@
+   ```
+
+3. Service name automatic detection
+
+   If no service name is explicitly configured one will be generated for you.
+   This can be helpful in some circumstances.
+
+   - If the application is hosted on IIS in .NET Framework this will be
+     `SiteName\VirtualPath` ex: `MySite\MyApp`
+   - If that is not the case it will use the name of the application
+     [entry Assembly](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getentryassembly?view=net-7.0).
+
+By default we recommend using environment variables for configuration. However,
+if given setting supports it, then:
+
+- use `Web.config` for configuring an ASP.NET application (.NET Framework),
+- use `App.config` for configuring a Windows Service (.NET Framework).
+
+## Global settings
+
+| Environment variable                 | Description                                                                                                                                                                                                                             | Default value | Status                                                    |
+| ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- | --------------------------------------------------------- |
+| `OTEL_DOTNET_AUTO_HOME`              | Installation location.                                                                                                                                                                                                                  |               | [Experimental](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES` | Names of the executable files that the profiler cannot instrument. Supports multiple comma-separated values, for example: `ReservedProcess.exe,powershell.exe`. If unset, the profiler attaches to all processes by default. \[1\]\[2\] |               | [Experimental](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_DOTNET_AUTO_FAIL_FAST_ENABLED` | Enables possibility to fail process when automatic instrumentation cannot be executed. It is designed for debugging purposes. It should not be used in production environment. \[1\]                                                    | `false`       | [Experimental](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_LOG_LEVEL`                     | SDK log level. (supported values: `none`,`error`,`warn`,`info`,`debug`)                                                                                                                                                                 | `info`        | [Stable](/docs/specs/otel/versioning-and-stability)       |
+
+\[1\] If `OTEL_DOTNET_AUTO_FAIL_FAST_ENABLED` is set to `true` then processes
```

Comment:
Not sure about those footnotes, there are footnotes supported by markdown but as far as I know they land at the very bottom of the page. Wondering if there are any that can be placed at the bottom of a section. Maybe something we can solve after merging, cc @chalin

Reply:
yeah, we can keep it and see later how to improve